lib/user_agent/browsers/acast.rb

```diff
@@ -0,0 +1,63 @@
+class UserAgent
+  module Browsers
+    # Acast/1.63.0 (Phone; iOS 11.2.6; iPad6,11)
+    # Acast/1.63.0 (Phone; iOS 11.2.5; iPhone9,4)
+    # Acast/1.42.1 (phone; android 10; Google Pixel 3 XL)
+    # Acast/1.26.0 (Linux;Android 9) ExoPlayerLib/2.2.0
+    class Acast < Base
+      ACAST_REGEX   = /Acast/
```

Comment:
```suggestion
      ACAST_REGEX   = /Acast/.freeze
```
I think you're gonna need to freeze these regex's here and for the other browsers even though rubocop claims you don't need to freeze it.